torchrec/csrc/dynamic_embedding/details/bits_op.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+#include <cstdint>
+#include <type_traits>
+
+namespace torchrec::dynamic_embedding::details {
```

Comment:
I'd say - no need for nested namespaces, just use torchrec namespace.

Reply:
fixed.